### DISCUSSION TITLE
Every proof I send to apply is rejected

### DISCUSSION THREAD

**Discussion by Franslike** [community/community#54871] [GitHub Education]
### Hi everyone,

since early today I have been trying to apply to the GitHub Global Campus from Venezuela, but once I start the process when they ask me for the proof, I send a photo of my recent university bills and they reject it, I try again and again it rejects it, claiming that I need to upload the photo of my student ID, however my university does not give this ID, it also gives me the message that I am not near a campus, however I am currently trying to apply from my job which is not far from my university campus, what can I do?

**Comment by coolacpc** [comment]
Welcome to the club 😔

**Reply by Franslike** [reply]
That's sad :( Is no other way to talk with someone who can help us with our situation?

**Reply by coolacpc** [reply]
yes it is. And no. No other solution, i have literally tried every single document NO LUCK! 😢

**Comment by samus-aran** [comment]
Hi @Franslike 👋 

In this particular case I would ask if you could try and re-applying again. This time you should receive more appropriate rejection reasons (if you receive any). Let us know how you get on 👍
